index.js

```diff
@@ -64,12 +66,15 @@ exports.deploy = async function (options) {
 };
 
 exports.callViewFunction = async function (options) {
+    checkCredentials(options);
```

Comment:
Don't need creds for a view function, eh?

Reply:
yep! forgot to remove this :)